packages/env/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+## 4.2.1 (2022-05-07)
+
+### Enhancement
+-   Added support for ssh protocol.
```

Comment:
```suggestion
### Enhancement
-   Added SSH protocol support for git sources
```

I'd mention something about git sources, and we also don't need to add the date (it gets automatically added by the release process, iirc :) )